src/components/calcite-pick-list-item/calcite-pick-list-item.scss

```diff
@@ -1,13 +1,17 @@
 :host {
-  @extend %component-host;
   @apply items-stretch
-    flex
-    text-color-1
-    shadow-border-bottom
-    m-0
-    mb-px;
+        flex
```

Comment:
This section needs the spacing/indentation updated.

I think there's something we run to clean up the code, no @driskull?

Reply:
Could you try the following?

1. run `npm install` – this should set up the project to lint and fix issues when committing
2. add a newline or space to this file
3. stage and commit the change locally

After this, the file should get formatted. If this is not the case, we'd need to look into what's preventing the linting from kicking in. Can you also make sure there's no custom `lint-staged` entry in `package.json`? This would override the existing configuration.

If the above fails and to move forward, you could run `npm run lint` and make sure to only commit the files relevant to this PR.

Reply:
@jcfranco I just get an error on commit. `[FAILED] Prevented an empty git commit!` and `npm run lint` didn't make any changes to files. I could just update those lines directly to be correctly tabbed.

Reply:
Nice. That means it's working. 🎉 

It seems Prettier preserves whitespace for this case. You can see this in action [here](https://prettier.io/playground/#N4Igxg9gdgLgprEAuZALCBnGACYAdKbbAAQEMAHcgGwE9sBLeAWwwFosAnOGMVAogYOwAzKnAAe-bPHExWkKhA6sAjFKIZUpACYQA7qwBGS7XGXGYMCE3VC7gpqwAMt+-aaHW5cQG4CUgMJpDlIoDEZ6aCRsQ1IwAGsAcw4IAFcobXkIRQ5sABIYELCI6D8g0PomUhhIqGiwUiowRjhWYR1W+kICovCa0qkAKlxbMkpaGIhxIxMzMqIAXylYhOS0jKyc6IA3Ug4AClZ5Rub4VlT6NqU4NfTMlQBKeZFoOXCALzgdvcPjpparrB2PRPkdHmUFiAADQgCDkfphZCgPYpPQABT2CAwyBAjT0pBo2JhhhCCW4AGVyHEuolkIVUnAYXAPHBtKZtAAZUKJVKkRJwABiSiqlhpONIqSs0JAqBgTCoAHVUC0MFSwHByViIttGDQcWAMESQF0MGYYGiQokqsh2lRTTCAFYYcQAIVJ8QppCYcA5XTgNsa9pATvE5JpYgAiqkIPAA3bGSAqRxTRwcRgDUbyBwujAFfRtDBUMgABxOGFZzBwBUhcg4rNwFPbf0wgCO0fgFrh2JQpDYUDgrNZ0q4bfoXAtfOtSFtQdNTHodI4DJh4SgiUj7f908DCZgpEMeYLRaQACYYYVSPQqDSAMLWKcgBsAVmlqVNABV992ZwntgyAJIZAgMDkmA2bwgAghk5IwDQYhxqaCwLEAA).

Reply:
Based on that, I'd manually tweak the space as it was for consistency.